editor/script_create_dialog.cpp

```diff
@@ -715,6 +720,7 @@ ScriptCreateDialog::ScriptCreateDialog() {
 	hb = memnew(HBoxContainer);
 	file_path = memnew(LineEdit);
 	file_path->connect("text_changed", this, "_path_changed");
+	file_path->connect("text_entered", this, "_path_entered");
```

Comment:
I think, instead of creating a new "_path_entered" function, you can connect it to the "_ok" function defined in the Dialog class.

Reply:
doesn't text_entered signal require string argument?

Reply:
error occurs if connect to `_ok`
```
ERROR: emit_signal: Error calling method from signal 'text_entered': 'ScriptCreateDialog::_ok': Method expected 0 arguments, but called with 1.
   At: core/object.cpp:1201.
```

Reply:
Oh true, I didn't see that :)